libs/event/src/lib/components/request-screening/request-screening.component.scss

```
@@ -1,4 +1,4 @@
-:host.animated button {
+:host.animated .pulse-button {
```

Isn't it possible to do
```suggestion
  this.animated = false
```
in `libs/event/src/lib/components/request-screening/request-screening.component.ts`